test/vcr_python_wrapper.py

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env python
+
+import os
+import sys
+import vcr
+import json
+
+VCR_PARAMS_FILE = '../../.vcr_params.json'
+
+sys.argv.pop(0)
+
+# Load recording parameters from file
+with open(VCR_PARAMS_FILE, 'r') as params_file:
+    test_params = json.load(params_file)
+cassette_file = 'fixtures/{}-{}.yml'.format(test_params['test_name'], test_params['serial'])
+# Increase serial and dump back to file
+test_params['serial'] += 1
+with open(VCR_PARAMS_FILE, 'w') as params_file:
+    json.dump(test_params, params_file)
+
+# Call the original python script with vcr-cassette in place
+with vcr.use_cassette(cassette_file,
+                      record_mode=test_params['record_mode'],
+                      match_on=['method', 'scheme', 'port', 'path', 'query']):
+    execfile(sys.argv[0])
```

Comment:
python3 removed execfile :anger:

Reply:
https://stackoverflow.com/a/16577427 suggests ```exec(open(sys.argv[0]).read())``` as an alternative.

Reply:
Or https://docs.python.org/3/library/runpy.html#runpy.run_path

Reply:
> For a simple script, the specified code is simply executed in a fresh module namespace.
Sonds like runpy creates some 'clean environment' the script runs in. That's exactly what we do not want. ;)